src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/Contracts/ExecutionPlanGraph.cs

```diff
@@ -127,6 +135,18 @@ public class ExecutionPlanGraphPropertyBase
         public PropertyValueDataType DataType { get; set; }
     }
 
+    public class ExecutionPlanCostMetrics
```

Comment:
Why is this object not passed?

Reply:
I meant to get rid of this. Originally I was using an array, but switched over to using a dictionary and no longer needed this. The dictionary is simpler to look things up in than having to sift through elements of an array to find the correct name/value pair.